nautobot/virtualization/templates/virtualization/vminterface.html

```diff
@@ -17,6 +17,12 @@
                     <td>Name</td>
                     <td>{{ object.name }}</td>
                 </tr>
+                <tr>
+                        <td>Status</td>
+                        <td>
+                            <span class="label" style="color: {{ object.status.color|fgcolor }}; background-color: #{{ object.status.color }}">{{ object.get_status_display }}</span>
```

Comment:
I'm a bit surprised we don't yet have a standard templatetag for rendering Status objects...

Reply:
I guess creating a template tag for rending the `Status` object is out of scope for this issue right?

Reply:
Yeah - but it would be good to open a housekeeping issue.